docs/data/material/components/snackbars/snackbars.md

```diff
@@ -139,3 +139,9 @@ The Snackbar component is composed of a root `<div>` that houses interior elemen
   </div>
 </div>
 ```
+
+## Experimental APIs
+
+### Imperative API
+
+You can create and manipulate notifications imperatively with the [`useNotifications`](https://mui.com/toolpad/core/react-use-notifications/) API in `@toolpad/core`. This API provides state management for opening and closing snackbars. It also allows for queueing multiple notifications at once.
```

Comment:
This felt a bit hard to grasp. I could see a live demo like we have above for notistack.

Reply:
We intentionally avoided adding code snippets that weren't "`@mui/material` native" to not confuse users. A demo without a snippet isn't as useful as the feature is in the API shape. Not sure this fully makes sense though.

Reply:
I was thinking of having the equivalent to https://mui.com/material-ui/react-snackbar/#notistack.

Reply:
Planning a refinement for the next iteration https://github.com/mui/mui-toolpad/issues/4007